CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/visual/InteractiveVKVisualProcessor.java

```diff
@@ -137,7 +137,7 @@ public GLSetHitTestingOperation(final boolean doHitTesting) {
 
         @Override
         public void apply() {
-            //setDrawHitTest(doHitTesting);
+            setDrawHitTest(doHitTesting);
```

Comment:
I don't think we need this at all.  We draw the hit test whenever the scene changes, not when some flag is set.